feat: change page title and add APS logo

AMAN081118 opened the pull request:
- **Updated Page Title**: Changed the default page title to match project branding requirements.
- **Added APS Favicon**: Included the APS logo as the favicon to enhance visual identity and consistency.

AMAN081118 committed:
feat: change page title and add APS logo

singh-ab commented:
> * **Updated Page Title**: Changed the default page title to match project branding requirements.
> 
>     * **Added APS Favicon**: Included the APS logo as the favicon to enhance visual identity and consistency.

The default page title is working fine now. But vercel/next.js logo appears in place of APS logo. Try looking into it.

AMAN081118 commented:
The issue has been fixed ✅. I’ve updated the page title and added the APS favicon, and both changes are now live on the deployed site.

Please note that the favicon may still display the default Vercel icon if cached by the browser. I recommend doing a hard refresh (or clearing the cache) to see the updated icon.

Attached is a screenshot showing the updated page title and favicon.

<img width="492" alt="aps_ss" src="https://github.com/user-attachments/assets/410b441a-1cf4-4401-a373-eda6c28b338d">

singh-ab commented:
I had tried loading it in different browsers and incognito/pvt window and it still didn't show up. But it appears to be working now so could've been an issue on vercel's side. Closing issue #54 now.
